oak_functions/client/java/BUILD

```diff
@@ -24,10 +24,9 @@ package(
 java_library(
     name = "client",
     srcs = [
-        "src/AeadEncryptor.java",
-        "src/AttestationClient.java",
-        "src/KeyNegotiator.java",
-        "src/Util.java",
+        "com/google/oak/functions/client/AeadEncryptor.java",
```

Comment:
Don't java projects normally have a `src` folder as well? (or would that violate some g3 file structure requirement?) I mean could not all this still be in a `src` folder, e.g., "src/com/google/oak/functions/client/AeadEncryptor.java"?

Reply:
Done